django_saml2_auth_lw/views.py

```diff
@@ -254,9 +254,9 @@ def signin(r):
 
     # Only permit signin requests where the next_url is a safe URL
     if parse_version(get_version()) >= parse_version('2.0'):
-        url_ok = is_safe_url(next_url, None)
+        url_ok = url_has_allowed_host_and_scheme(next_url, None)
     else:
-        url_ok = is_safe_url(next_url)
+        url_ok = url_has_allowed_host_and_scheme(next_url)
```

Comment:
yeah looks good to me. Referenced with https://github.com/django/django/blob/88ed1c8d08c70fd3e7943fc8383459545f726dcd/django/utils/http.py#L270-L276